### PR TITLE
Updating translations - v3.5

### PR DESCRIPTION
**Pull request by horus68**
Updating translations for packed languages - version 3.5

Details
-------
Global translation updates with latest work from Crowdin

All languages files has passed by v3.5 plain install to order strings

### CODE REVIEW

**Comment by SvanteRichter** [comment]
Does this not remove some translations too? Or are those no longer used? Like for example

 * markdown, licence and prefill from infos.cs_CZ.yml
 * markdown from infos.fi.yml
 * markdown and video from infos.ja.yml
 * basically everything from infos.nb.yml

(just the first examples that popped out when skimming the diff)

**Comment by horus68** [comment]
As far as I'm aware this includes all translations done in GIT but with modifications from several users done via Crowdin https://crowdin.com/project/bolt 
Some users had directly sent their translations to Github as single PR and I have also included it on Crowdin for some time.
This PR do not includes strings with english values and that is why it looks some files are wiped out.
All language files had been tested with bolt 3.5 and it works (language strings are ordered by Bolt too after Admin / Configuration / Messages).

True, it looks like a big clean up but this is because strings are reorder each time they pass by Bolt as "translated/non translated/not in use)

On your examples:
- infos.nb.yml = the actual file in GIThub is only english, not translated
https://github.com/bolt/bolt/blob/3.5/app/resources/translations/nb/infos.nb.yml

- On the fields markdown, short_help, video and license there is an error that I'm gone to fix. This was caused by a change done sometime ago in the source strings from  `&amp;quot;` to `&quot; `  and also with `&lt;embed&gt;`
This caused the translations to end up in pending state for revision. I'm revising it now and will update the commit for all languages

**Review by GwendolenLynch — CHANGES_REQUESTED** [review]
Needs work. Flagging as such.

**Comment by GwendolenLynch** [comment]
For reference, we standardise on 4 spaces for YAML indentation, so if you could adjust whichever tool you're using to output in that format it would be greatly appreciated.

File: app/resources/translations/cs_CZ/messages.cs_CZ.yml [hunk 1]
```diff
@@ -7,75 +7,411 @@
...
+"Add some sample <a href='%url%' class='btn btn-default'>Records with Loripsum text</a>": "Přidat ukázku <a href='%url%' class='btn btn-default'>Záznam s Lorem ipsum</a>"
+"Current (saved) status:": "Současný (uložený) stav:"
+"Do you really want to delete %FOLDERNAME%?": "Opravdu smazat %FOLDERNAME%?"
+"Excerpt": "Souhrn"
+"Expand sidebar": "Roztáhnout postranní lištu"
+"The settings for roles and permissions are stored in <code>%permissions%</code>.": "Nastavení uživatelský práv a rolí je v <code>%permissions%</code>."
+"Title": "Nadpis"
+"View (saved version) on site": "Zobrazit uloženou verzi na webu"
+"contenttypes.generic.actions-all": "Akce pro %contenttypes%"
+"contenttypes.generic.actions-one": "Akce pro %contenttype%"
```
Comment (GwendolenLynch) [inline app/resources/translations/cs_CZ/messages.cs_CZ.yml:31]: Dotted keywords shouldn't be quoted. Is this coming out of the tool you're using?

Comment (horus68) [reply app/resources/translations/cs_CZ/messages.cs_CZ.yml]: Dotted keywords? I didn't change the strings. I can see in the github or in the download those:

```
"You've been logged on successfully.": "You've been logged on successfully."
contenttypes.generic.actions-all: "Actions for %contenttypes%"
```

Comment (horus68) [reply app/resources/translations/cs_CZ/messages.cs_CZ.yml:31]: @GawainLynch it was not me, but the Bolt lint tool. All "Unused messages" messages gets the keyword quoted after been passed by the language lint tool from Bolt (Admin / config / messages)

Example: Strings in the downloaded file for cs_CZ:

```
"Title": "Nadpis"
"Everybody": "Všichni"
users.your-password: "Vaše heslo"
contenttypes.generic.group.taxonomy: "Taxonomie"
contenttypes.generic.group.template: "Šablona"

```

Strings after been saved by BOLT:
```
#--- Unused messages ----------------------------------------------------------

"Title": "Nadpis"
"users.your-password": "Vaše heslo"


#--- Legacy translation messages ----------------------------------------------

"Everybody": "Všichni"

#--- Translation messages -----------------------------------------------------

contenttypes.generic.group.taxonomy: "Taxonomie"
contenttypes.generic.group.template: "Šablona"

```
As you can see (in `users.your-password` ) the Unused and Legacy messages are quoted by Bolt. The "translations section are not edited by BOLT.
So this shouldn't be a problem to block this PR but a bug alert to solve in a different PR.

**Comment by horus68** [comment]
@SahAssar changes requested by you are done.

**Comment by horus68** [comment]
@GawainLynch on the 4 spaces for YAML indentation I can't change the behavior from Crowdin (the YAML files are directly downloaded from them).
I will file a bug on them as when downloading the source file its fine (no spaces changes), only the translations have half the spaces on indentations.

**Comment by GwendolenLynch** [comment]
> @GawainLynch on the 4 spaces for YAML indentation I can't change the behavior from Crowdin (the YAML files are directly downloaded from them).

Then Crowdin is not a suitable tool to use, as I mentioned a while ago as a reason for _not_ using it.

**Pull request closed without merging**